### ISSUE TITLE
AccessibilityManager.destroy() fails if called twice

### ISSUE THREAD

**Issue by Cristy94** [pixijs/pixi.js#5176]
Your issue may already be reported! Please search [existing issues](../) before creating one.

### Expected Behavior
Calling `app.renderer.plugins.accessibility.destroy();` after the game is created shouldn't result in an error when the game is destroyed.

### Current Behavior
If you destroy the accessibility manager, the game will throw an error when it is destroyed, as the function tries to access `this.children` which was set to null.

### Possible Solution
http://pixijs.download/dev/docs/accessibility_AccessibilityManager.js.html#line518

```
    /**
     * Destroys the accessibility manager
     *
     */
    destroy()
    {

       // Don't try to destroy again if already destroyed
       if (this.children === null) { return; } // <--- ADD THIS

        this.div = null;
        for (let i = 0; i < this.children.length; i++)
        {
            this.children[i].div = null;
        }
        window.document.removeEventListener('mousemove', this._onMouseMove, true);
        window.removeEventListener('keydown', this._onKeyDown);
        this.pool = null;
        this.children = null;
        this.renderer = null;
    }
````

- `pixi.js` version: _e.g. 4.7.3_

**Comment by bigtimebuddy** [comment]
This came up recently in another issue that I could find. If you manually destroy the manager you need to delete from the plugins:

app.renderer.plugins.accessibility.destroy();
delete app.renderer.plugins.accessibility;
